kvmagent/kvmagent/plugins/vms/vm_host_file.py

```
@@ -0,0 +1,195 @@
+def write_vm_host_file(to):
+    # type: (VmHostFileTO) -> None
+    if not is_allowed_paths(to.path):
+        to.error = "%s is not in allowed path" % to.path
+        return
+    if not to.contentBase64:
+        return
+    
+    try:
+        # raw_data is str in python 2.7
+        raw_data = base64.b64decode(to.contentBase64) # type: str
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)